assets/package.json

```diff
@@ -15,8 +15,11 @@
   "devDependencies": {
     "@babel/cli": "7.23.4",
     "@babel/core": "7.23.7",
+    "@babel/plugin-syntax-import-meta": "^7.10.4",
     "@babel/preset-env": "7.23.8",
     "css.escape": "^1.5.1",
+    "esbuild": "^0.23.1",
+    "esbuild-plugin-wasm": "^1.1.0",
```

Comment:
I think I don't need these anymore. I was struggling with wasm and esbuild but found another work around.